Sources/CircularProgress/util.swift

```diff
@@ -45,19 +45,21 @@ extension NSColor {
 		return HSBAColor(Double(hue), Double(saturation), Double(brightness), Double(alpha))
 	}
 
-	private func colorWithSaturation(ratio: Double, brightness: Double? = nil) -> NSColor {
+	/// Adjust color components by ratio.
+	func adjusting(
```

Comment:
Suggestions welcome for a better name.

Reply:
I think it’s alright. I considered `adjusted`, like [`reversed`](
https://developer.apple.com/documentation/swift/array/1690025-reversed), but that doesn’t sound right, does it?

Reply:
I'm still very much unsure when to use `ed` and when to use `ing` for non-mutating methods...

This is what the Swift API design guidelines says:

> When the operation is naturally described by a verb, use the verb’s imperative for the mutating method and apply the “ed” or “ing” suffix to name its nonmutating counterpart.

> Prefer to name the nonmutating variant using the verb’s past participle (usually appending “ed”):

> When adding “ed” is not grammatical because the verb has a direct object, name the nonmutating variant using the verb’s present participle, by appending “ing.”

https://swift.org/documentation/api-design-guidelines/#name-according-to-side-effects

The thing is, Swift has methods like `appending()`, but `appended()` is also a valid verb, so they don't seem to be following their own guidelines...

Reply:
I see what you mean. In that case I wouldn't worry too much about it. To me `adjusting` sounds equally wrong and right  as `adjusted` and then I prefer `adjusting`. 😉